src/main/java/com/minecolonies/coremod/colony/ColonyView.java

```diff
@@ -662,4 +663,16 @@ public int getLastContactInHours()
     {
         return lastContactInHours;
     }
+
+    @Override
+    public boolean hasWillRaidTonight()
+    {
+        return false;
+    }
+
+    @Override
+    public List<EntityLivingBase> getGuardTargets()
+    {
+        return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1168)